src/main/java/io/buildpacks/example/sample/PeerReview.java

```diff
@@ -1,7 +1,5 @@
-// PeerReview.java
-
```

Comment:
2. In the main method, add comments to explain the purpose of each test case to make it easier to understand the expected output.